chart/values.yaml

```diff
@@ -25,6 +25,13 @@ controller:
       enabled: false
       tokenFilepath: ""
       mirrorSlackChannelId: ""
+    http:
+      enabled: true
+      url: "http://notifier.chaos-demo.svc.cluster.local:8080"
+      headers:
+        - "key:value"
```

Comment:
I think the values in `chart/values.yaml` should be good defaults, rather than examples